.github/workflows/ci.yml

```diff
@@ -166,6 +166,129 @@ jobs:
           path: |
             ~/.m2/repository/org/apache/ozone
           retention-days: 1
+  build-config-doc:
+    needs:
+      - build
+    if: ${{ github.repository == 'apache/ozone' && github.event_name == 'push' && github.ref_name == 'master' }}
```

Comment:
We can relax this condition to allow running for any branch in forks.  Hard-coded references to `apache` should be replaced with current repo's owner.  The job(s) should handle gracefully (skip without error) the case when the owner does not have `ozone-site` repo.  PRs should not need to be created in forks.  This would allow testing most of the steps without manually editing the workflow.